homeassistant/core.py

```diff
@@ -610,8 +610,9 @@ def async_add_job[_R](
     @callback
     def async_add_job[_R, *_Ts](
         self,
-        target: Callable[[*_Ts], Coroutine[Any, Any, _R] | _R]
-        | Coroutine[Any, Any, _R],
+        target: (
+            Callable[[*_Ts], Coroutine[Any, Any, _R] | _R] | Coroutine[Any, Any, _R]
+        ),
```

Comment:
This looks like an unrelated formatting change ?

Reply:
Ah, that was probably Black auto-formatting the file because I had it open in IntelliJ. My bad.